contrib/templates/scala-job/template/build.sbt.tmpl

```diff
@@ -0,0 +1,21 @@
+scalaVersion := "{{template `scala_version` .}}"
+
+name := "{{.project_name}}"
+organization := "com.examples"
+version := "{{template `version` .}}"
+
+// If we are using 16.2, we will fetch it locally since as of 14-02-2025, we have not released scala client 16.2.0.
+// Otherwise we will fetch it from remote maven repo.
+resolvers += "Local Maven" at "file://" + new File("./local_dependencies").getAbsolutePath
+libraryDependencies += "com.databricks" % "databricks-connect" % "{{template `dbr_version` .}}.+"
+libraryDependencies += "org.slf4j" % "slf4j-simple" % "2.0.16"
+
+assembly / assemblyOption ~= { _.withIncludeScala(false) }
+assembly / assemblyExcludedJars := {
+  val cp = (assembly / fullClasspath).value
+  cp filter { _.data.getName.matches("scala-.*") } // remove Scala libraries
+}
+
```

Comment:
What do you think about adding this option as an option into `sbt run`.

Reply:
sorry I was not clear. I meant adding the "--add-opens" option as a JVM parameter for sbt run.

https://www.scala-sbt.org/1.x/docs/Forking.html#Forked+JVM+options